pyatlan/model/assets/dbt_column_process.py

```diff
@@ -570,6 +588,7 @@ class Attributes(Dbt.Attributes):
 
 
 from .airflow_task import AirflowTask  # noqa
+from .alteryx_workflow import AlteryxWorkflow  # noqa
```

Comment:
This is a one-off customization (in-progress experiment) — we should remove from generated code.